### PR TITLE
Add Tilt to Angle

### PR DESCRIPTION
**Pull request by bplett-wgtcorp**
"Tilt" is commonly used in Tilt Meters, and it is simply the Sin of the angle.

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/AngleTests.cs [hunk 1]
```diff
@@ -38,6 +38,8 @@ public class AngleTests : AngleTestsBase
 
         protected override double RevolutionsInOneDegree => 2.777777777777777e-3;
 
+        protected override double TiltInOneDegree => 0.01745240643728351281941897851632;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/AngleTests.cs:41]: Could you point me to a source reference where I can verify this, besides your calculation? I can't seem to find any.

I found these:
https://en.wikipedia.org/wiki/Inclinometer
https://en.wikipedia.org/wiki/Grade_(slope)

**Comment by bplett-wgtcorp** [comment]
Verifying the constant is trivial.  In fact, had you not previously advised me against it, I wouldn’t have used a constant.  I’d just have used:
protected override double TiltInOneDegree => Math.Sin(RadiansInOneDegree);

Hope that helps!
Brad.

File: UnitsNet.Tests/CustomCode/AngleTests.cs (hunk 1 above)
Comment (angularsen) [reply UnitsNet.Tests/CustomCode/AngleTests.cs:41]: @bplett-wgtcorp Right, but what I meant is that I can't find any source of reference that says the measurement "tilt" equals Sin(radians) of an angle. A wiki or unit converter or any kind of digital source would be helpful to verify this unit.

**Comment by bplett-wgtcorp** [comment]
Ahh, sorry.  I actually get lots of “hits”.  Try this search:  https://www.google.com/search?q=%2Btilt+meter+output+%2Bsine+angle

Using just the top three on the results I get:
From https://www.geosense.co.uk/media/BlockAttributeValueFile/152/file/2TiltMeterBeamApplicationGuideV1.0.pdf I get “The readings generated by Digital Tilt Sensors are in ‘Sine of the Angle’.”
From https://www.encardio.com/blog/tiltmeter-how-does-it-work-installation-how-to-read/ I get “The tilt sensor thus provides a bipolar DC voltage output proportional to the sine of tilt angle measured by the tiltmeter.”
From https://www.althensensors.com/landing-pages/tilt-angle-measurement/ I get “…provides an output signal of +/- 5 volts proportional to the sine wave”

I have nothing to do with any of those companies.  😊  Is that sufficient?  Would you like more?

Thanks!
Brad.

File: UnitsNet.Tests/CustomCode/AngleTests.cs (hunk 1 above)
Comment (angularsen) [reply UnitsNet.Tests/CustomCode/AngleTests.cs:41]: @bplett-wgtcorp Thanks, no those are plenty!

**Comment by angularsen** [comment]
Thanks!
[Release UnitsNet/4.88.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.88.0)

**Comment by bplett-wgtcorp** [comment]
Fantastic!  Thanks a lot!

Brad.

**Pull request merged**